eof/download.py

```diff
@@ -107,22 +91,56 @@ def download_eofs(orbit_dts=None, missions=None, sentinel_file=None, save_dir=".
             query.update(client.query_orbit_for_product(sentinel_file))
         else:
             for mission, dt in zip(missions, orbit_dts):
-                result = client.query_orbit(dt, dt + timedelta(days=1),
-                                            mission, product_type='AUX_POEORB')
+                found_result = False
+                products = client.query_orbit(dt - ScihubGnssClient.T0,
+                                              dt + ScihubGnssClient.T1,
+                                              mission,
+                                              product_type='AUX_POEORB')
+                result = (client._select_orbit(products, dt, dt + timedelta(minutes=1))
+                          if products else None)
                 if result:
+                    found_result = True
                     query.update(result)
                 else:
                     # try with RESORB
-                    result = client.query_orbit(dt, dt + timedelta(minutes=1),
-                                                mission,
-                                                product_type='AUX_RESORB')
-                    query.update(result)
+                    products = client.query_orbit(dt - timedelta(hours=1),
```

Comment:
without the +/- 1 hour window for restituted orbits, I couldn't ever get the sentinelsat API to return any results